ffs/filcold/filcold.go

```diff
@@ -262,6 +262,9 @@ func (fc *FilCold) WaitForDeals(ctx context.Context, c cid.Cid, proposals []cid.
 	}
 
 	if len(activeProposals) == 0 {
+		if ctx.Err() != nil {
+			return nil, errors, fmt.Errorf("all accepted proposals were untracked due to cancellation")
+		}
```

Comment:
Better error message if making deas failed but because Powergate is shutting down and things got canceled in purpose.